src/profile.command.ts

```diff
@@ -32,20 +32,33 @@ export abstract class ProfileCommand extends Command {
         this.spinner.setSpinnerString('|/-\\');
     }
 
-    public getProfile(options: ProfileOptions): Profile {
-        const profileName = options.profile ? options.profile : 'default';
+    public getProfile(): Profile {
         try {
-            return this.profileService.findProfileNamed(profileName);
+            return this.profileService.getCurrentProfile();
```

Comment:
If the user passes the option --profile to a command, the profile name passed should be taken as the default profile instead of the profile set as current profile.